### PR TITLE
Proposed update

### PR DESCRIPTION
**Pull request by tbrowder**

### CODE REVIEW

**Comment by 2colours** [comment]
@tbrowder some links got rewritten by URL's of your fork.

**Comment by tbrowder** [comment]
Yes, I see. I'll correct that.

**Comment by tbrowder** [comment]
My changes should cause the test urls to be correct when the README is regenerated in the community repository.

**Comment by 2colours** [comment]
@tbrowder you outsourced the readme into its own file. I don't have a strong opinion about that but in this case, you should probably remove the same content from the file of the module.

**Comment by tbrowder** [comment]
Okay, I removed that pod from lib source, and I also got File::Find working for Windows

File: dist.ini [hunk 1]
```diff
@@ -1,13 +1,19 @@
-name = File-Find
+name = File::Find
```
Comment (2colours) [inline dist.ini:1]: What is this used for? On one hand, the name of the released distribution is already `File::Find` but on the other hand, the folder of the repo better stays `File-Find` in my opinion.

File: README.md [hunk 2]
```diff
@@ -29,32 +46,32 @@ Specify a name of the file `File::Find` is ought to look for. If you pass a stri
...
 =================
 
-Please note, that this module is not trying to be the verbatim port of Perl's File::Find module. Its interface is closer to Perl's File::Find::Rule, and its features are planned to be similar one day.
+Please note that this module is not trying to be the verbatim port of Perl's File::Find module. Its interface is closer to Perl's File::Find::Rule, and its features are planned to be similar one day.
 
 CAVEATS
 =======
 
-List assignment is eager in Raku, so if You assign `find()` result to an array, the elements will be copied and the laziness will be spoiled. For a proper lazy list, use either binding (`:=`) or assign a result to a scalar value (see SYNOPSIS).
+List assignment is eager in Raku, so if you assign `find()` result to an array, the elements will be copied and the laziness will be spoiled. For a proper lazy list, assign a result to a scalar value (see SYNOPSIS).
```
Comment (2colours) [inline README.md:76]: Any objections against binding?

File: README.md [hunk 3]
```diff
@@ -29,32 +46,32 @@ Specify a name of the file `File::Find` is ought to look for. If you pass a stri
...
 keep-going
 ----------
 
 Parameter `keep-going` tells `find()` to not stop finding files on errors such as 'Access is denied', but rather ignore the errors and keep going.
 
-recursive
----------
-
-By default, `find` will recursively traverse a directory tree, descending into any subdirectories it finds. This behaviour can be changed by setting `recursive` to a false value. In this case, only the first level entries will be processed.
+**Note: This parameter is currently broken and does not affect the search in any way.**
```
Comment (2colours) [inline README.md:66]: It does affect the search. If I remember correctly, we have cleared this up. The test cases might be broken but it basically does the right thing nevertheless.

**Comment by 2colours** [comment]
I had a couple of comments to negotiate but overall I like the direction of the changes. 👍

File: README.md (hunk 2 above)
Comment (tbrowder) [reply README.md:76]: Not I.

Comment (2colours) [reply README.md:76]: I meant "use either binding (`:=`) or" disappeared from the code. (The capital y was probably a "polonism")

File: dist.ini (hunk 1 above)
Comment (tbrowder) [reply dist.ini:1]: That is for 'mi6' use. I think it will work either way, but the "real" module package name is best.

Comment (2colours) [reply dist.ini:1]: Fair enough. Seems like it's used for figuring out the "main module" so why not.

File: README.md (hunk 3 above)
Comment (tbrowder) [reply README.md:66]: This note has been removed.

File: README.md (hunk 2 above)
Comment (tbrowder) [reply README.md:76]: Corrected to original wording.

File: README.md [hunk 4]
```diff
@@ -29,32 +46,32 @@ Specify a name of the file `File::Find` is ought to look for. If you pass a stri
...
+
+By default, `find` will recursively traverse a directory tree, descending into any subdirectories it finds. This behavior can be changed by setting `recursive` to a false value. In this case, only the first-level entries will be processed.
 
 keep-going
 ----------
 
 Parameter `keep-going` tells `find()` to not stop finding files on errors such as 'Access is denied', but rather ignore the errors and keep going.
 
-recursive
----------
```
Comment (2colours) [inline README.md:47]: Was this removed intentionally now?

File: README.md (hunk 3 above)
Comment (2colours) [reply README.md:66]: I still see it - didn't you accidentally remove something else?

File: README.md (hunk 4 above)
Comment (2colours) [reply README.md:47]: Oh nevermind, it just moved somewhere else.

File: README.md (hunk 3 above)
Comment (tbrowder) [reply README.md:66]: This is getting hard to read. I'm going to close this PR and rebubmit

**Comment by tbrowder** [comment]
Closing. Will resubmit.

**Pull request closed without merging**